### DISCUSSION TITLE
Where's the Question Node in Dialogic 2.0?

### DISCUSSION THREAD

**Discussion by caduceusj** [dialogic-godot/dialogic#1645] [Q&A] [answered]
![image](https://github.com/coppolaemilio/dialogic/assets/85528178/c1d077ca-414c-4f11-a0ee-1b8c23690e60)

I ve just installed the plugin and i can't find this question node anywhere, please help

**Comment by KoB-Kirito** [comment, marked as answer]
It is no more. Just add choices and the text event before them will be used as the question automatically.
